### PR TITLE
Auth pages

### PR DESCRIPTION
**Pull request by shubhmehra**
Updates in auth pages

### CODE REVIEW

File: components/auth/login.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="stylesheet" href="/style.css" />
```
Comment (devdeadviz) [inline components/auth/login.html:8]: Just a suggestion, try to use relative path convention.

File: components/auth/login.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  </head>
+  <body>
+    <div class="main-container">
+      <main class="home-container">
+        <div id="main-navbar" class="ab-navbar-shadow"></div>
+        <div class="ab-auth-wrapper">
+          <div class="card ab-auth-card">
+            <div class="card-title">Login</div>
+            <div
+              class="card-text ab-auth-card-body ab-display-flex ab-flex-flow-column"
```
Comment (devdeadviz) [inline components/auth/login.html:19]: Try to add comments for the codebase which is not generic to another user.

File: components/auth/login.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            >
+              <div class="input-group">
+                <label>Username/Email *</label>
+                <input
+                  type="email"
+                  class="au-input"
```
Comment (devdeadviz) [inline components/auth/login.html:25]: Use self-declarative naming convention instead of shorthands.

File: style.css [hunk 4]
```diff
@@ -2,7 +2,7 @@
 @import url("/components/cart/cart.css");
 @import url("/components/wishlist/wishlist.css");
 @import url("/components/product-list/product-list.css");
-@import url("/components/login/login.css");
+@import url("/components/auth/auth.css");
 
 .main-container-width {
   max-width: 1200px;
```
Comment (devdeadviz) [inline style.css]: Try to use relative units instead of absolute units.

**Review by devdeadviz — CHANGES_REQUESTED** [review]
Try to update suggested changes!

**Comment by devdeadviz** [comment]
Add proper description about what you are adding in this pull request along with a gif or screenshots.

File: components/auth/login.html (hunk 3 above)
Comment (shubhmehra) [reply components/auth/login.html:25]: AU is for reference of my component library AdornUI which I've used here as helper class.
I will add proper comments & docs for same.

File: components/auth/login.html (hunk 2 above)
Comment (shubhmehra) [reply components/auth/login.html:19]: Ok.. 
I will add that same thing in documentation so other user can understand that.

File: components/auth/login.html (hunk 1 above)
Comment (shubhmehra) [reply components/auth/login.html:8]: Ok..

File: style.css (hunk 4 above)
Comment (shubhmehra) [reply style.css]: It is main container width so I've tried to give it an intrinsic value

**Pull request merged**